account_payment_term_extension/readme/CONFIGURE.md

```diff
@@ -0,0 +1,19 @@
+To configure the Payment Terms and see the new options on the Payment
+Term Lines, you need to:
+
+1.  Go to the menu Invoicing \> Configuration \> Invoicing \> Payment
+    Terms.
+
+To use multiple payment days, define for each payment term line which
+payment days apply, separated by spaces, commas or dashes. To use
+holidays, insert the holiday and the date payment terms will be
+postponed to.
+
```

Comment:
I think an explanation of how to configure Percent (Untaxed amount) should be added